spec/services/course_search_service_spec.rb

```diff
@@ -7,19 +7,23 @@
       let(:filter) { {} }
 
       it "defaults to Course" do
-        expect(Course).to receive_message_chain(:findable, :distinct).and_return(findable_scope)
-        expect(subject).to eq(findable_scope)
+        expect(Course).to receive_message_chain(:findable).and_return(course_ids_scope)
```

Comment:
As we are not longer calling `distinct` there are lots of changes in this spec. I'm not a huge fan of RSpec 'shared examples' but perhaps there is a case for using them here given the repetition.

Reply:
you probably could of left it as as it `scope` and the `outer_scope` does the ordering etc now...